resources/fixtures/Clay_Paky/Clay-Paky-Sharpy.qxf

```diff
@@ -102,16 +102,10 @@
  </Channel>
  <Channel Name="Prism Rotation">
   <Group Byte="0">Prism</Group>
-  <Capability Min="0" Max="20">Position 0°</Capability>
-  <Capability Min="21" Max="41">Position 90°</Capability>
-  <Capability Min="42" Max="62">Position 180°</Capability>
-  <Capability Min="63" Max="83">Position 270°</Capability>
-  <Capability Min="84" Max="104">Position 360°</Capability>
-  <Capability Min="105" Max="126">Position 450°</Capability>
-  <Capability Min="127" Max="127">Position 540°</Capability>
-  <Capability Min="128" Max="190">Rotation (Fast 43rpm to slow 1.1rph)</Capability>
-  <Capability Min="191" Max="192">Stop</Capability>
-  <Capability Min="193" Max="255">Rotation (Slow 1.1rph to fast 43rpm)</Capability>
+  <Capability Min="0" Max="127" Preset="RotationIndexed">Rotation Indexed</Capability>
```

Comment:
Why do you take the liberty to destroy a channel like this??
Revert

Reply:
The list of exact rotation positions (90°, 180°, 270°) for particular DMX values is a clumsy way to define the indexed rotation of a gobo. We have a preset for this: "RotationIndexed". Therefore we replaced the custom capabilities with a preset. We can revert this to being useless if you like.

Reply:
![image](https://user-images.githubusercontent.com/2951337/107225940-c3678480-6a19-11eb-960c-2bfb197936f7.png)

First, this is prism and not gobo. Second, this is not a "clumsy way". You intentionally fucked up a channel that was clearly defined in the product manual. See screenshot.

Reply:
OK, our mistake. We will revert this and all similar changes. Please answer my [general comment blow](https://github.com/mcallegari/qlcplus/pull/1266#issuecomment-774985431).